build.gradle

```diff
@@ -125,16 +145,38 @@ jacocoTestReport {
   }
 }
 
-project.tasks['sonarqube'].dependsOn jacocoTestReport
+project.tasks['sonarqube'].dependsOn test, integration, jacocoTestReport
+check.dependsOn integration
+checkstyleMain.shouldRunAfter(compileJava)
+test.shouldRunAfter(checkstyleTest)
+integration.shouldRunAfter(checkstyleIntegrationTest)
+integration.shouldRunAfter(test)
 
 sonarqube {
   properties {
-    property "sonar.projectName", "Reform :: spring-boot-template"
-    property "sonar.projectKey", "uk.gov.hmcts.reform:spring-boot-template"
+    property "sonar.projectName", "SSCS - Hearings Api"
+    property "sonar.projectKey", "SSCSHA"
     property "sonar.coverage.jacoco.xmlReportPaths", "${jacocoTestReport.reports.xml.destination.path}"
+    property "sonar.pitest.mode", "reuseReport"
+    property "sonar.pitest.reportsDirectory", "build/reports/pitest"
+    property "sonar.exclusions", "src/main/java/uk/gov/hmcts/reform/sscs/Application.java" +
+      "src/main/java/uk/gov/hmcts/reform/sscs/config/**,"
   }
 }
+pitest {
+  junit5PluginVersion = '0.12'
+  targetClasses = ['uk.gov.hmcts.reform.sscs.*']
+  excludedClasses = [
+    'uk.gov.hmcts.reform.sscs.config.*',
+    'uk.gov.hmcts.reform.sscs.Application'
```

Comment:
Change to  uk.gov.hmcts.reform.sscs.HearingApiApplication

Reply:
Done